.github/workflows/python-ci-full.yml

```diff
@@ -5,7 +5,9 @@ name: TileDB-SOMA Python CI (Full)
 # releases.
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+      - 'release-**'
```

Comment:
Is two stars here intentional?

Reply:
@thetorpedodog just googlecopypasta ...

Reply:
@thetorpedodog the thing I googlecopypastaed from may have been wanting to follow branches that have slashes in them -- I think I saw `release/**` -- since we don't use slashes in our release-branch names, I think a single star would suffice